### DISCUSSION TITLE
column layout problem with mac os safari + github pages

### DISCUSSION THREAD

**Discussion by won-j** [mmistakes/minimal-mistakes#2859] [Q&A]
New to Jekyll, I created a GitHub page using the Minimal Mistakes [remote theme starter](https://github.com/mmistakes/mm-github-pages-starter/generate) to see how it looks. Files were *not* modified *at all* from starter template. The page was published from the master branch. I also tested the page layout by applying the [local method](https://docs.github.com/en/github/working-with-github-pages/testing-your-github-pages-site-locally-with-jekyll) to the local clone of the repo.

On the Safari in my MacBook pro, the two versions appear quite differently.

## Environment

The following is the environment for the local testing. The Github page uses the remote theme (as-is the starter template); MM version not specified).

- Minimal Mistakes version: 4.22.0
- Ruby gem or remote theme version: 2.7.2
- Jekyll version: 3.9.0
- Git repository URL: https://github.com/won-j/won-j.github.io
- GitHub Pages hosted (if yes provide URL to site): https://won-j.github.io
- Operating system: mac os catalina 10.15.7
- Safari: version 14.0 (15610.1.28.1.9, 15610)

## Expected behavior

Ordinary two-column layout as shown below:
<img width="1024" alt="local" src="https://user-images.githubusercontent.com/15611033/109843427-62d10d80-7c8e-11eb-826d-4d6e4d5ffcac.png">

Local testing gives consistent layouts as above whether  they are shown in Safari or Chrome. 

## Steps to reproduce the behavior

However, if I visit the published pages using Safari, the left column disappears and the content is shown as a banner near the top of the page. Links to Twitter or Github are condensed into a button:

<img width="1024" alt="github" src="https://user-images.githubusercontent.com/15611033/109845493-90b75180-7c90-11eb-9530-3fe084b3e9a3.png">

If I visit  this page using Chrome (on the same MacBook pro) the appearance are exactly the same as the local ones. Mobile Safari from my iPad pro 13-inch gave the ordinary result. 

The "banner version" appears to be a mobile version: when I visit this page from my iPhone using mobile Safari, I see the same banner and button. (It is also interesting since in my iPad pro Safari was set to a mobile version.)

**Comment by mmistakes** [comment]
I can't reproduce. Seems like something with your browsers on your device or setup. Perhaps there's some sort of content blocker installed. It appears that it's trying to show the smaller viewport version (mobile) where the side bar is collapsed and has the Follow button.

When I test your site on my Mac in Safari and Chrome it looks as it should. Same for my iPad.

![Screen Shot 2021-03-04 at 10 26 03 AM](https://user-images.githubusercontent.com/1376749/109986752-098bcd00-7cd4-11eb-8f6b-7f85fb0a1753.png)
![Screen Shot 2021-03-04 at 10 27 47 AM](https://user-images.githubusercontent.com/1376749/109987052-4952b480-7cd4-11eb-80b1-6995374fb193.png)
